Resources/Textures/_Harmony/Clothing/Head/Hats/black_bow.rsi/meta.json

```
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "drawn by Svetoslav228",
```

You should specify (GitHub) so they know what platform the username refers to.
```suggestion
  "copyright": "Drawn by Svetoslav228 (GitHub)",
```